e2e/azure-node-e2e/tests/azure-node.spec.ts

```diff
@@ -10,14 +11,14 @@ describe('azure-node e2e', () => {
     jest.setTimeout(300_000);
 
     beforeAll(async () => {
-        await newProject('@ensono-stacks/azure-node', ['@nx/next']);
+       await newProject(['@ensono-stacks/next', '@ensono-stacks/azure-node'], ['@nx/next']);
```

Comment:
Now that we are accepting multiple packages within `newProject` we must also ensure that the implicitDependencies are updated as currently this will install `@ensono-stacks/next@latest` from npm, rather than a version built from the codebase and published to verdaccio. 

In the `azure-node-e2e/package.json` file we will need to add next to the implicitDependencies array. This will ensure that when the e2e executor runs, it will build and publish a version of the next package to verdaccio. 

- [ ] Update package.json
- [ ] Add a new e2e section to the contribution guide explaining this requirement